h/views/admin/groups.py

```diff
@@ -46,24 +46,32 @@ def groups_index(context, request):
     renderer="h:templates/admin/groups_create.html.jinja2",
     permission="admin_groups",
 )
-class GroupCreateController(object):
+class GroupCreateView(object):
+    """View handlers for creating new groups"""
```

Comment:
```suggestion
     """Views for the admin create group form."""
```

They're just called "views" (or sometimes "view callables" or I guess view methods or view functions) in Pyramid. "View handler" means [something else](https://docs.pylonsproject.org/projects/pyramid_handlers/en/latest/).